identity/app/views/verificationEmailResent.scala.html

```diff
@@ -1,51 +1,43 @@
 )(implicit request: RequestHeader, context: model.ApplicationContext)
 
-@import common.LinkTo
-
-@emailBlock = {
-    <div class="identity-forms-email-wrap">
-        <header>@user.primaryEmailAddress</header>
-        <aside>
-            Not the right email? <a href="@idUrlBuilder.buildUrl(controllers.editprofile.routes.EditProfileController.redirectToAccountSettings.url, idRequest)" data-link-name="mma : verify-email : update-email-from-error">Change it here</a>
-        </aside>
-    </div>
-}
-
 <div class="identity-wrapper monocolumn-wrapper">
     <section class="identity-forms-message">
-        @if(isSignupFlow) {
-            <h1 class="identity-title">Please check your inbox</h1>
-        } else {
-            <h1 class="identity-title">To continue, you must confirm this is your email address</h1>
-        }
-        @emailBlock
+        <h1 class="identity-title">Please verify your email to complete your registration</h1>
         <div class="identity-forms-message__body">
-            @if(isSignupFlow) {
-                <p>We've sent you an email – please open it up and click on the verify button to confirm your email address.</p>
-            } else {
-                <p>We have sent a link to this email address. Please check your inbox and follow the instructions.</p>
+            @email match {
+                case Some(email) => {
+                    <p>We've sent a verification email to:</p>
+                    <p><b>@email</b></p>
+                }
+                case _ => {
+                    <p>We've sent you a verification email</p>
+                }
             }
+            <p>Please click on the link in this email to confirm your email address and complete your registration.</p>
         </div>
         <aside class="identity-forms-message__body">
-            <p class="identity-forms-message__explainer">Note that the link is only valid for 30 minutes, so be sure to open it soon!</p>
-            <a class="manage-account__button manage-account__button--center js-id-send-validation-email" data-link-name="mma : verify-email : resent">Resend confirmation email</a>
+            <p class="identity-forms-message__explainer">Note that the link is only valid for 30 minutes. If you can't find it, it may be in your spam folder.</p>
         </aside>
-        @if(isSignupFlow) {
-            <footer class="identity-forms-message__body">
-                <a class="u-underline" href="@LinkTo {@verifiedReturnUrl.getOrElse(defaultReturnUrl)}" data-link-name="mma : verify-email : exit-to-gu">
-                    @(verifiedReturnUrl match {
-                        case Some(_) => "Exit and continue"
-                        case None => "Exit and go to The Guardian home page"
-                    })
-                </a>
-            </footer>
-        }
+        <footer class="identity-forms-message__body">
+            @validationEmailResent match {
+                case Some(emailResent) => {
```

Comment:
Instead of using Option[Boolean] with Some(false) indicating the error scenario, it would be more clear to use a sealed trait with 3 case objects.